### PR TITLE
Swagger answers endpoit

### LINKED ISSUE #30: Backend: Endpoint to retrieve answers - GET api/answers
## User story

**Who:** **As a** developer
**What:** **I want to** be able to get answers with specific criteria
**Why:** **so that we can** show those answers

## Implementation details
- Create the `GET` endpoint for `/api/answers`

**Think about ways to filter the questions that make sense. These should be added as query parameters. Ask if this does not make sense**

Remember to split the story into smaller PR's! Maybe you can do a PR with just the words, filter fx. You can also choose to do it in separate commits. just be very aware of the PR's not becoming too big!

Check the README file here https://github.com/HackYourFuture-CPH/guide-it#-working-with-api-documentation---swagger and this article should also help you with that https://blog.logrocket.com/documenting-your-express-api-with-swagger/

### PR DESCRIPTION
**Pull request by Enqira**
# Description

Created answers endpoint /api/answers with swagger.
Opctions: GET answers, GET answers by id, DELETE answer.

Not sure about the DELETE one. Reviewers let us know if it should be descarted
Fixes #30 

# How to test?
To test it, go to  api/documentations then test it using swagger.
# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by TITHI1244 — APPROVED** [review]
As per my understanding, the issue is only about get endpoint for answers table. But the code looks nice :)

**Review by bytewiz — CHANGES_REQUESTED** [review]
Looks good one small thing :v:

File: src/server/api/controllers/answers.controller.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  }
+
+  try {
+    const answer = await knex('answers')
+      .select('answers.id as id', 'answer')
+      .where({ id });
+    if (answer.length === 0) {
+      throw new Error(`incorrect entry with the id of ${id}`, 404);
+    }
+    return answer;
```
Comment (bytewiz) [inline src/server/api/controllers/answers.controller.js:20]: This still returns an array is that what you would expect using the endpoint? :)

**Review by bytewiz — DISMISSED** [review]

File: src/server/api/controllers/answers.controller.js (hunk 1 above)
Comment (Enqira) [reply src/server/api/controllers/answers.controller.js]: Thanks Niels, I made the requested changes to return just an object instead of an array :)

**Comment by Enqira** [comment]
Emplemented requested changes by Niels. and it is ready to be merged

**Review by dpfernandes — APPROVED** [review]
👍

**Review by bytewiz — APPROVED** [review]

**Pull request merged**